### DISCUSSION TITLE
Why doesn't findFirst offer the "null" type?

### DISCUSSION THREAD

**Discussion by yepMad** [prisma/prisma#11228] [Q&A]
Hi!

I'm new in Prisma. Why do **findFirst** type definitions assume that the returned value will **always be valid**? Why isn't it said that it can return **"null"**?

![image](https://user-images.githubusercontent.com/22564368/149848620-371b6daf-ba13-4bcb-9799-d5dfccdeb81a.png)

**Comment by austincrim** [comment]
Hey @yepMad 👋 ,

`findFirst` does indeed have a chance of returning `null`. Per engineering, this discrepancy is because some of our type utilities require the `"strictNullChecks": true` tsconfig setting.

In any case, would you mind creating a [bug report](https://github.com/prisma/prisma/issues/new?assignees=&labels=kind%2Fbug&template=bug_report.yml) with this info? That way our product and engineering teams won't lose track of it and we might be able to improve down the road.

Thanks!

**Reply by vladi-stevanovic** [reply]
👋 Hello @yepMad I'm going through pending GitHub Discussions to check if you need any help. Is this issue still open?
Please let us know if we can investigate it further or if you've already solved it, we'd love to know how!

It would be super useful for other users facing this problem! :raised_hands:

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
